src/Core/Domain/OrderState/Command/AddOrderStateCommand.php

```diff
@@ -0,0 +1,231 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\OrderState\Command;
+
+use PrestaShop\PrestaShop\Core\Domain\OrderState\Exception\OrderStateConstraintException;
+
+/**
+ * Adds new order state with provided data
+ */
+class AddOrderStateCommand
+{
+    /**
+     * @var string[]
+     */
+    private $localizedNames;
+    /**
+     * @var string
+     */
+    private $color;
+    /**
+     * @var bool
+     */
+    private $logable;
+    /**
+     * @var bool
+     */
+    private $invoice;
+    /**
+     * @var bool
+     */
+    private $hidden;
+    /**
+     * @var bool
+     */
+    private $sendEmail;
+    /**
+     * @var bool
+     */
+    private $pdfInvoice;
+    /**
+     * @var bool
+     */
+    private $pdfDelivery;
+    /**
+     * @var bool
+     */
+    private $shipped;
+    /**
+     * @var bool
+     */
+    private $paid;
+    /**
+     * @var bool
+     */
+    private $delivery;
+    /**
+     * @var array
+     */
+    private $localizedTemplates;
+
+    /**
+     * @param string[] $localizedNames
+     * @param string[] $localizedTemplates
+     */
+    public function __construct(
+        array $localizedNames,
+        string $color,
+        bool $logable,
```

Comment:
```suggestion
        bool $loggable,
```

Please fix this misspelling (every where (methods))